packages/lit-dev-content/site/docs/ssr/server-usage.md

```diff
@@ -8,6 +8,116 @@ eleventyNavigation:
 
 {% labs-disclaimer %}
 
+## Rendering templates
+
+Server rendering begins with rendering a Lit _template_ with a server-specific `render()` function provided in the `@lit-labs/ssr` package.
+
+The siganture of the render function is:
+
+```ts
+render(value: unknown, renderInfo?: Partial<RenderInfo>): RenderResult
+```
+
+Typically `value` is a TemplateResult produced by a Lit template expression, like:
+
+```ts
+html`<h1>Hello</h1>`
+```
+
+The template can contain custom elements, which are rendered in turn, along with their templates.
+
+```ts
+import {render} from '@lit-labs/ssr';
+
+const result = render(html`
+  <h1>Hello SSR!</h1>
+  <my-element></my-element>
+`);
+```
+
+To render an single element, you render a template that only contains that element:
+
+```ts
+const result = render(html`<my-element></my-element>`);
+```
+
+### Handling RenderResults
+
+`render()` returns a `RenderResult`: an iterable of values that can be streamed or concatenated into a string.
+
+A `RenderResult` can contain strings, nested `RenderResult`s, or Promises of strings or `RenderResult`s. Not all RenderResults will contain Promises - those can occur when custom elements preform async tasks, like fetching data - but because it can contain Promises, processing a RenderResult into a string or an HTTP response is _potentially_ an async operation.
```

Comment:
```suggestion
A `RenderResult` can contain strings, nested `RenderResult`s, or Promises of strings or `RenderResult`s. Not all RenderResults will contain Promises - those can occur when custom elements preform async tasks, like fetching data - but because it can contain Promises, processing a `RenderResult` into a string or an HTTP response is _potentially_ an async operation.
```